mmv1/templates/terraform/examples/uptime_check_config_cloud_run.erb

```diff
@@ -0,0 +1,33 @@
+resource "google_monitoring_uptime_check_config" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
To fix the error: the compiler is expecting this file to be `uptime_check_config_cloud_run.tf.erb`

Reply:
Sorry about that, I updated in this commit : https://github.com/GoogleCloudPlatform/magic-modules/pull/10820/commits/525c522998edeed41ccefca3db1ab10151dee6ff

Thanks for your advice!